packages/features/form-builder/FormBuilder.tsx

```diff
@@ -0,0 +1,734 @@
+                required
+                isDisabled={
+                  fieldForm.getValues("editable") === "system" ||
+                  fieldForm.getValues("editable") === "system-but-optional"
+                }
+                onChange={(e) => {
+                  const value = e?.value;
+                  if (!value) {
+                    return;
+                  }
+                  fieldForm.setValue("type", value);
+                }}
+                value={FieldTypes.find((fieldType) => fieldType.value === fieldForm.getValues("type"))}
+                options={FieldTypes.filter((f) => !f.systemOnly)}
+                label="Input Type"
+              />
+              <InputField
+                required
+                {...fieldForm.register("name")}
+                containerClassName="mt-6"
+                disabled={
+                  fieldForm.getValues("editable") === "system" ||
+                  fieldForm.getValues("editable") === "system-but-optional"
+                }
+                label="Name"
+              />
+              <InputField
+                {...fieldForm.register("label")}
+                // System fields have a defaultLabel, so there a label is not required
+                required={!["system", "system-but-optional"].includes(fieldForm.getValues("editable") || "")}
+                placeholder={t(fieldForm.getValues("defaultLabel") || "")}
+                containerClassName="mt-6"
+                label="Label"
+              />
+              {fieldType?.isTextType ? (
+                <InputField
+                  {...fieldForm.register("placeholder")}
+                  containerClassName="mt-6"
+                  label="Placeholder"
+                  placeholder={t(fieldForm.getValues("defaultPlaceholder") || "")}
+                />
+              ) : null}
+
+              {fieldType?.needsOptions ? (
+                <Controller
+                  name="options"
+                  render={({ field: { value, onChange } }) => {
+                    return <OptionsField onChange={onChange} value={value} className="mt-6" />;
+                  }}
+                />
+              ) : null}
+              <Controller
+                name="required"
+                control={fieldForm.control}
+                render={({ field: { value, onChange } }) => {
+                  return (
+                    <BooleanToggleGroupField
+                      disabled={fieldForm.getValues("editable") === "system"}
+                      value={value}
+                      onValueChange={(val) => {
+                        onChange(val);
+                      }}
+                      label="Required"
+                    />
+                  );
+                }}
+              />
+              <DialogFooter>
+                <DialogClose color="secondary">Cancel</DialogClose>
+                <Button type="submit">{isFieldEditMode ? t("save") : t("add")}</Button>
```

Comment:
I can't edit fields. When clicking save this error is thrown:
![Screenshot 2023-02-15 at 10 04 40](https://user-images.githubusercontent.com/30310907/219066670-3520a631-4b2c-4a0f-b4ff-10d6f7d96377.png)

Reply:
Ahh..regression due to recent name conflict check. Fixed it.